### PR TITLE
Add <mountedPart> and <parts> elements

### PR DESCRIPTION
**Pull request by silviooliva**
"mountedPart" and "parts" elements have been introduced to describe the "parts", i.e. the devices
other than the MCUs (e.g. sensors, connectivity devices, ...).
When mounted on a board, a part can be described into a board pack using the
"mountedPart" element.
The "parts" element can be used instead to describe the "parts" into part packs.

Signed-off-by: Silvio Lucio Oliva <silvio.oliva@st.com>

### CODE REVIEW

**Comment by JonatanAntoni** [comment]
Does this change correspond to some issue or requirement under discussion? Who else than Arm-staff is expected to review and comment?

**Comment by silviooliva** [comment]
> Does this change correspond to some issue or requirement under discussion? 

It corresponds to issue #144 

> Who else than Arm-staff is expected to review and comment?
I guess NXP.

File: doxygen/src/pdsc_format.txt [hunk 1]
```diff
@@ -72,6 +72,7 @@ The \ref packFormat is structured using grouping elements and contains the follo
  - \subpage pdsc_devices_pg "<devices>": lists the devices supported by this \ref cp_SWComponents "Software Pack". It specifies the device attributes and
    files like flash programming algorithms, CMSIS device header files, CMSIS System View Descriptions, etc.
  - \subpage pdsc_boards_pg "<boards>": lists the development boards supported by the \ref cp_SWComponents "Software Pack".
+ - \subpage pdsc_parts_pg "<parts>": lists the parts (devices other than the MCUs) supported by the \ref cp_SWComponents "Software Pack".
```
Comment (ReinhardKeil) [inline doxygen/src/pdsc_format.txt:75]:
```suggestion
 - \subpage pdsc_parts_pg "<parts>": lists the parts (devices other than the MCUs) supported by the \ref cp_SWComponents "Software Components".
```

**Comment by jkrech** [comment]
@silviooliva - All checks have passed now. There is nothing else for you to do at this point. #144 request for approval request email was sent out to the Decision Committee last night. My approval is pending on that. Thanks a lot for your patience.

**Comment by silviooliva** [comment]
> All checks have passed now. There is nothing else for you to do at this point. #144 request for approval request email was sent out to the Decision Committee last night. My approval is pending on that. Thanks a lot for your patience.

Thank you, @jkrech @ReinhardKeil @JonatanAntoni.

**Review by jkrech — APPROVED** [review]
Looks good to me. The corresponding change request also now got approved.

**Pull request merged**